examples/Serilog.Sinks.ElmahIo.Example/Program.cs

```diff
@@ -46,6 +50,9 @@ static void Main(string[] args)
             logger.Information("A message with {type} {hostname} {application} {user} {source} {method} {version} {url} and {statusCode}",
                 "custom type", "custom hostname", "custom application", "custom user", "custom source", "custom method",
                 "custom version", "custom url", 500);
+
+            Console.WriteLine("Wait 5 seconds and press any key to exit.");
+            Console.ReadKey();
```

Comment:
Maybe add an explicit `logger.CloseAndFlush()` ?

Reply:
Good idea. Probably the solution to fixing the broken tests as well. Thanks!

Reply:
`logger.CloseAndFlush` isn't available and `Log.CloseAndFlush` doesn't seem to trigger `EmitBatch`. Any idea of what could be wrong here?

Reply:
strange: https://github.com/serilog/serilog/wiki/Lifecycle-of-Loggers Should be the trick.

Reply:
Ah, you're right. Either use `Log.Logger` or wrap the entire thing in a `using`. Thanks.